nixos/modules/system/boot/systemd/initrd.nix

```diff
@@ -135,6 +127,13 @@ let
   };
 
 in {
+  options.boot.initrd.environment.etc = mkOption {
+    description = "Set of files that have to be linked in /etc";
+    visible = false;
+    default = {};
+    type = options.environment.etc.type;
```

Comment:
```suggestion
    type = options.environment.etc.type;
    visible = false;
    default = {};
    description = "Set of files that have to be linked in /etc";
```

Reply:
This is going to need some tweaking anyway. The `type = options.environment.etc.type` trick is dirty and for some reason it doesn't work when evaluating the manual.